test/node/object_id.test.ts

```diff
@@ -5,6 +5,13 @@ import { expect } from 'chai';
 import { bufferFromHexArray } from './tools/utils';
 import { isBufferOrUint8Array } from './tools/utils';
 
+declare module '../register-bson' {
+  interface ObjectId {
+    pool: Uint8Array;
+    offset: number;
+  }
+}
```

Comment:
Should be able to remove this, if you were expirencing type errors it's because register-bson depends on the types being built, so if you switch branches etc, `npm run build:dts` (or just `npm install`) is a prerequiste to get the current library's types.